pkg/replication/replication_mode.go

```
@@ -232,46 +232,44 @@ func (m *ModeManager) loadDRAutoSync() error {
 	return nil
 }
 
-func (m *ModeManager) drSwitchToAsyncWait(availableStores []uint64) error {
+func (m *ModeManager) drSwitchToAsyncWait(availableStores []uint64) {
```

Why change it?

---

The caller always ignored this error, and all errors were logged within the function, so I removed it.

---

But the tests will check the return value of `drSwitchToAsync`.

---

Got it. I will revert it for the test